ux-of-data-fetching/Exercise1/solution/.codesandbox/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM node:20-bullseye
```

Comment:
I think CodeSandbox uses Node 16 by default but this exercise requires Node 18+.